doc: Add Milesight EM300 TH (#1246)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/models/milesight-em300-th/_index.md

---
title: "Milesight EM300-TH"
description: ""
weight: 
---

{{< figure src="milesight-EM300-TH.png" alt="Milesight EM300-TH" class="float plain" width="75%" >}}

The [Milesight EM300-TH](https://www.milesight.com/iot/product/lorawan-sensor/em300-th) is an environment sensor suitable for monitoring indoor and outdoor temperature and humidity. It can be widely used in office areas, greenhouses and other environments.

<!--more-->

## Provisioning Information

> The provisioning information, **DevEUI**, can be found on the back of the device. You can use `24E124C0002A0001` as the **JoinEUI**. The **AppKey** can be auto-generated within {{% tts %}}.

Reviewer: ```suggestion
The provisioning information can be found in various places:
The **DevEUI** is located on the back of the device. 
The **AppEUI/JoinEUI** is `24E124C0002A0001` by default.
The **AppKey** can be found inside of the email when purchasing device.
```

This is a rough suggestion, word it how you like, but the Appkey shouldnt be auto-generated, it is a value given with the device.

Author: @LDannijs Thanks for the info.

---
title: "Milesight EM300-TH"
description: ""
weight: 
---

{{< figure src="milesight-EM300-TH.png" alt="Milesight EM300-TH" class="float plain" width="75%" >}}

The [Milesight EM300-TH](https://www.milesight.com/iot/product/lorawan-sensor/em300-th) is an environment sensor suitable for monitoring indoor and outdoor temperature and humidity. It can be widely used in office areas, greenhouses and other environments.

<!--more-->

> ## Provisioning Information

Reviewer 2: ```suggestion
## Provisioning Information

```

The provisioning information, **DevEUI**, can be found on the back of the device. You can use `24E124C0002A0001` as the **JoinEUI**. The **AppKey** can be auto-generated within {{% tts %}}.

> 

Reviewer 2: ```suggestion
```

{{< figure src="milesight-EM300-TH-back.png" alt="EM300-TH provisioning information" >}}

> 

Reviewer 2: ```suggestion
```

> {{< note "The Milesight EM300-TH doesn’t have a QR code that supports {{% tts %}} to extract provisioning information." />}}

Reviewer 2: ```suggestion
{{< note "The Milesight EM300-TH cannot be onboarded to {{% tts %}} using QR code." />}}
```

Reviewer 2: Does this device have a QR code? But that QR code cannot be used for onboarding to TTS? I'm asking because if there's no QR code at all, you can remove this part completely, but if there is (but cannot be used to onboard to TT), then you can simplify this as I suggested

Author: It has a QR code but doesn't support onboarding.
